images/win/toolsets/toolset-2022.json

```diff
@@ -257,7 +257,8 @@
         ],
         "vsix": [
             "VisualStudioClient.MicrosoftVisualStudio2022InstallerProjects",
-            "WixToolset.WixToolsetVisualStudio2022Extension"
+            "WixToolset.WixToolsetVisualStudio2022Extension",
```

Comment:
you probably don't want this commit to be here? :)

Reply:
Let's cancel this one also. I will create new PR for both Windows and MacOS.